views/partials/_header.njk

```diff
@@ -1,3 +1,5 @@
+
+{% from "label/macro.njk" import govukLabel %}
```

Comment:
Given we're not using any of the styling from the component, does it still make sense to use the macro? Is there any benefit over just using a `<label>`?